cricketscircle/jekyll-test
# Image galery

## What: 
allows user to upload images and see current gallery
## How to test:
on line 47 and 48 in js/main.js you need to provide access key and secret key, ask me or Bryce. 
go to /cms/ click on New Story and then View Image Gallery. Drag and drop images. Hit Upload and you should see those images in the gallery list at the bottom.
